streamPair.q

```diff
@@ -0,0 +1,80 @@
+// load tables
+tab1: 40000#1_ flip `dateTime`bid`ask`bidVol`askVol!("*FFFF";",") 0: `:C:/q/dash/sample/data/stocks/USA500IDXUSD.csv;

```

Comment:
Where does this `40000` come from?

Reply:
I take only 40000 registers so that both tables have the same size. Otherwise they cause problems when calculating beta and alpha

Reply:
Okay, we should definitely obtain historical EOD (End of Day) data and use that for fitting Linear Regression, instead of doing this.